<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RUOK</class>
 <widget class="QWidget" name="RUOK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>311</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="img2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>331</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="img3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>30</y>
     <width>341</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="img1_comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9.jpg</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择待修改人脸:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>420</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择目标人脸:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>420</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>修改后人脸</string>
   </property>
  </widget>
  <widget class="QComboBox" name="img2_comboBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8.jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9.jpg</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="widget_pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>570</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>进入必做界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_pushButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>540</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>生成图像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>480</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择插值方式:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="method_comboBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>530</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>最近邻</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双线性</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双三次</string>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="lambda_horizontalSlider">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>610</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>580</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择正则化参数大小:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
